SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialTraceEventBuilder.h

```diff
@@ -0,0 +1,216 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "Interop/Connection/SpatialTraceEvent.h"
+#include "SpatialCommonTypes.h"
+#include "WorkerSDK/improbable/c_worker.h"
+
+#define GDK_EVENT_NAMESPACE "unreal_gdk."
+
+namespace SpatialGDK
+{
+class FSpatialTraceEventBuilder
+{
+public:
+	FSpatialTraceEventBuilder(const char* InType);
+	FSpatialTraceEventBuilder(const char* InType, const FString& InMessage);
```

Comment:
Same nit as above - take this by value and move the result internally. In most cases you are calling this with a literal so this will remove a copy.